stdlib/list.mli

```diff
@@ -359,6 +359,44 @@ val filteri : (int -> 'a -> bool) -> 'a list -> 'a list
    @since 4.11
 *)
 
+
+(** {1 List manipulation} *)
+
+
+val take : int -> 'a list -> 'a list
+(** [take n l] returns the prefix of [l] of length [n],
+    or [l] itself if [n > length l].
```

Comment:
```suggestion
    or [l] itself if [n > length l].
```
This is not, strictly speaking, true: it is not `l` that is returned, but a _copy_ of `l`. The implementation could be tweaked to make the comment true, though.

Reply:
```suggestion
    or a copy of [l] if [n > length l].
```
Let’s fix the documentation then. I’m not sure it’s worth the niche optimization. If someone thinks it is, I can revert and change the code.